testapi.pm

```diff
@@ -1187,6 +1189,8 @@ sub save_tmp_file {
     open my $fh, ">", $path;
     print $fh $content;
     close $fh;
+
+    return $path;
```

Comment:
Would you mind adding a simple test for this? `t/99-full-stack.t` already covers it, so you can probably just add a check for the returned path there

Reply:
How about modifying t/data/tests/tests/modify_and_upload_file.pm ?

Reply:
The test API functions are generally tested in `t/03-testapi.t`.